content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -764,7 +764,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
   multiple kube-apiservers exist at varied versions.
   See [Mixed version proxy](/docs/concepts/architecture/mixed-version-proxy/) for more information.
-- `UserNamespacesStatelessPodsSupport`: Enable user namespace support for stateless Pods.
+- `UserNamespacesSupport`: Enable user namespace support for Pods.
```

Comment:
This is a good place to mention the renaming.

Reply:
Did it, thanks!